projects/default/Makefile

```diff
@@ -44,11 +34,11 @@ endif
 clean: $(TARGETS)
 	@-rm -rf controllers/sumo_supervisor/*.pyc
 
-controllers/%.Makefile libraries/%.Makefile: libraries.Makefile controllers/ros/include.Makefile
+controllers/%.Makefile libraries/%.Makefile: $(ROS_DEPENDENCIES)
```

Comment:
This is not sufficient in case of make clean, as after a make clean (or cleanse) I still have (on Linux):
```
# testing if everything was cleansed...
Supprimerait projects/default/libraries/ros/libapr-1.so.0
Supprimerait projects/default/libraries/ros/libaprutil-1.so.0
Supprimerait projects/default/libraries/ros/libboost_filesystem.so.1.58.0
Supprimerait projects/default/libraries/ros/libboost_regex.so.1.58.0
Supprimerait projects/default/libraries/ros/libboost_system.so.1.58.0
Supprimerait projects/default/libraries/ros/libboost_thread.so.1.58.0
Supprimerait projects/default/libraries/ros/libconsole_bridge.so.0.2
Supprimerait projects/default/libraries/ros/libcpp_common.so
Supprimerait projects/default/libraries/ros/libicudata.so.55
Supprimerait projects/default/libraries/ros/libicui18n.so.55
Supprimerait projects/default/libraries/ros/libicuuc.so.55
Supprimerait projects/default/libraries/ros/liblog4cxx.so.10
Supprimerait projects/default/libraries/ros/librosconsole.so
Supprimerait projects/default/libraries/ros/librosconsole_backend_interface.so
Supprimerait projects/default/libraries/ros/librosconsole_log4cxx.so
Supprimerait projects/default/libraries/ros/libroscpp.so
Supprimerait projects/default/libraries/ros/libroscpp_serialization.so
Supprimerait projects/default/libraries/ros/librostime.so
Supprimerait projects/default/libraries/ros/libxmlrpcpp.so
Supprimerait projects/default/libraries/ros_kinetic_ubuntu16.04.tar.gz
```